bower.json

```diff
@@ -16,16 +16,16 @@
     "output"
   ],
   "dependencies": {
-    "purescript-transformers": "^4.0.0",
-    "purescript-tuples": "^5.0.0",
-    "purescript-lists": "^5.0.0",
-    "purescript-tailrec": "^4.0.0",
-    "purescript-mmorph": "^5.0.0",
-    "purescript-prelude": "^4.0.0",
-    "purescript-aff": "^5.0.0"
+    "purescript-transformers": "^5.0.0",
+    "purescript-tuples": "^6.0.0",
+    "purescript-lists": "^6.0.0",
+    "purescript-tailrec": "^5.0.0",
+    "purescript-mmorph": "git://github.com/fsoikin/purescript-mmorph#purescript-0.14",
```

Comment:
This temporarily points to https://github.com/Thimoteus/purescript-mmorph/pull/11 so that Bower picks up new dependencies for `mmorph`.

Reply:
Changed to `master`